test/test.js

```diff
@@ -24,6 +24,8 @@ const { Connection } = require('../lib/rpc/connection');
 const { helper } = require('../lib/helper');
 const { BrowserTypeDispatcher } = require('../lib/rpc/server/browserTypeDispatcher');
 
+Error.stackTraceLimit = 15;
```

Comment:
Why this?

Reply:
That's because you like to assert stack against __filename.